src/offlinelogstorage/dlt_offline_logstorage.h

```diff
@@ -53,28 +53,30 @@
 
 #include <search.h>
 #include <stdbool.h>
+#include <zlib.h>
```

Comment:
Hi, quick question regarding zlib use in dlt-deamon. If none of `WITH_DLT_LOGSTORAGE_GZIP`, `WITH_DLT_COREDUMPHANDLER`, or `WITH_DLT_FILETRANSFER` are defined, isn't `ZLIB_LIBRARY` undefined? And if so, how does CMake find this header (assuming it's not installed on the system), and how is zlib linked to dlt-daemon?

I'm trying to understand how this project works without zlib installed on the system already (if at all). Thanks!

Reply:
Some of this might be obsolete now since this PR is old and AFAIC see there has been some re-writing of logstorage. ZLIB is linked with the daemon in `src/daemon/CMakeLists.txt`. Guarded by the variables you mentioned.
If ZLIB is excluded the variable is set to *empty string*.